react/features/chat/functions.ts

```diff
@@ -93,14 +95,11 @@ export function getUnreadCount(state: IState) {
     }
 
     let reactionMessages = 0;
-
-    if (!lastReadMessage) {
```

Comment:
This will break the jump to last new message feature on web, won't it? What is this supposed to do?

Reply:
This was added after conversion to typescript so we can tackle  

Argument of type 'IMessage | undefined' is not assignable to parameter of type 'IMessage'.   Type 'undefined' is not assignable to type 'IMessage'.